Unable to produce a ConcreteFunction with two outputs

When making a signature with two outputs, I get the following error:

```
[error] (run-main-10) org.tensorflow.exceptions.TensorFlowException: Function instantiation included invalid output index: 1 not in [0, 1).
[error] org.tensorflow.exceptions.TensorFlowException: Function instantiation included invalid output index: 1 not in [0, 1).
[error]         at org.tensorflow.internal.c_api.AbstractTF_Status.throwExceptionIfNotOK(AbstractTF_Status.java:101)
[error]         at org.tensorflow.GraphOperationBuilder.finish(GraphOperationBuilder.java:415)
[error]         at org.tensorflow.GraphOperationBuilder.build(GraphOperationBuilder.java:88)
[error]         at org.tensorflow.GraphOperationBuilder.build(GraphOperationBuilder.java:67)
[error]         at org.tensorflow.op.core.StatelessPartitionedCall.create(StatelessPartitionedCall.java:94)
[error]         at org.tensorflow.op.core.PartitionedCall.create(PartitionedCall.java:65)
[error]         at org.tensorflow.ConcreteFunction.call(ConcreteFunction.java:244)
[error]         at org.tensorflow.op.core.Function.call(Function.java:42)
[error]         at org.tensorflow.op.Ops.call(Ops.java:1214)
[error]         at org.tensorflow.ConcreteFunction.call(ConcreteFunction.java:327)
```

I believe the issue is that here the code claims to be asking for output data types, but it passes the input data types:

https://github.com/tensorflow/java/blob/50f7efa5821877668c2645886b0e4addef7306b6/tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java#L247

see the call site expects output data types:
https://github.com/tensorflow/java/blob/50f7efa5821877668c2645886b0e4addef7306b6/tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/op/core/PartitionedCall.java#L42

this is with a snapshot version of 0.4.0 on August 3, 2021.

Here is an example that also throws an error related to wrong number of outputs, but not the exact same string:
```scala
    def dbl(tf: Ops) = {                                               
      val x: Placeholder[TInt32] = tf.placeholder(classOf[TInt32]);    
      val dblX = tf.math.add(x, x);    
      val tripX = tf.math.add(x, dblX)    
      Signature.builder().input("x", x).output("dbl", dblX).output("tripX", tripX).build();             
    }                                                                     
                                                                       
    val cf = ConcreteFunction.create(new Function[Ops, Signature] {    
      def apply(o: Ops) = dbl(o)    
    })                                                               
                                                                                       
    val out = cf.call(Map("x" -> (TInt32.scalarOf(2): Tensor)).asJava)    
    println(out.get("dbl").asInstanceOf[TInt32].getInt())                                    
    println(out.get("tripX").asInstanceOf[TInt32].getInt())  
```

which prints:
```
2021-08-03 16:19:19.303110: F external/org_tensorflow/tensorflow/core/framework/op_kernel.cc:973] Check failed: index < outputs_.size() (1 vs. 1)
zsh: abort
```

Well, I tried to build on (on macos), but I got a cryptic error:
```
[INFO] TensorFlow Java Parent ............................. SUCCESS [  0.099 s]
[INFO] TensorFlow Core Parent ............................. SUCCESS [  0.001 s]
[INFO] TensorFlow Core Generators ......................... SUCCESS [  0.538 s]
[INFO] TensorFlow Core API Library ........................ FAILURE [  0.237 s]
[INFO] TensorFlow Core API Library Platform ............... SKIPPED
[INFO] TensorFlow Framework Library ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.982 s
[INFO] Finished at: 2021-08-03T16:39:41-10:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.bytedeco:javacpp:1.5.4:build (javacpp-build) on project tensorflow-core-api: Execution javacpp-build of goal org.bytedeco:javacpp:1.5.4:build failed: Process exited with an error: 127 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :tensorflow-core-api
```

I'm not very experienced with mvn.

I tried to build with your command but it failed because the javadoc seems super broken:
```
[ERROR] Error fetching link: /Users/oboykin/oss_code/tfjava/tensorflow-framework/target/javadoc-bundle-options. Ignored it.                                                 
[ERROR] Error fetching link: /Users/oboykin/oss_code/tfjava/tensorflow-core/tensorflow-core-platform/target/apidocs. Ignored it.                                            
[INFO] No previous run data found, generating javadoc.                                                                                                                      
[INFO]                                                                                                                                                                      
100 errors                                                                                                                                                                  
[INFO] ------------------------------------------------------------------------                                                                                             
[INFO] Reactor Summary for TensorFlow Java Parent 0.4.0-SNAPSHOT:                                                                                                           
[INFO]                                                                                                                                                                      
[INFO] TensorFlow Java Parent ............................. SUCCESS [ 44.571 s]                                                                                             
[INFO] TensorFlow Core Parent ............................. SUCCESS [  0.016 s]                                                                                             
[INFO] TensorFlow Core Generators ......................... SUCCESS [ 27.917 s]                                                                                             
[INFO] TensorFlow Core API Library ........................ SUCCESS [02:55 min]                                                                                             
[INFO] TensorFlow Core API Library Platform ............... SUCCESS [ 54.179 s]                                                                                             
[INFO] TensorFlow Framework Library ....................... FAILURE [05:57 min]                                                                                             
[INFO] ------------------------------------------------------------------------                                                                                             
[INFO] BUILD FAILURE                                                                                                                                                        
[INFO] ------------------------------------------------------------------------                                                                                             
[INFO] Total time:  11:00 min                                                                                                                                               
[INFO] Finished at: 2021-08-03T17:04:37-10:00                                                                                                                               
[INFO] ------------------------------------------------------------------------                                                                                             
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (attach-javadocs) on project tensorflow-framework: MavenReportException: Error while 
generating Javadoc:                                                                                                                                                         
[ERROR] Exit code: 1 - /Users/oboykin/oss_code/tfjava/tensorflow-framework/src/main/java/org/tensorflow/framework/activations/SELU.java:17: error: cannot find symbol
[ERROR] import org.tensorflow.Operand;                                                                                                                                      
[ERROR]                      ^                                                                                                                                              
[ERROR]   symbol:   class Operand                                                     
[ERROR]   location: package org.tensorflow                                                                                                                                  
[ERROR] /Users/oboykin/oss_code/tfjava/tensorflow-framework/src/main/java/org/tensorflow/framework/activations/SELU.java:18: error: package org.tensorflow.op does not exist
[ERROR] import org.tensorflow.op.Ops;                                                                                                                                       
[ERROR]                         ^              

... (many more)
```

--- a/tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java
+++ b/tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java
@@ -244,7 +244,7 @@ public class ConcreteFunction implements AutoCloseable, TensorFunction {
         PartitionedCall.create(
                 scope,
                 inputList,
-                Arrays.stream(inputDtypes)
+                Arrays.stream(outputDtypes)
                     .map(x -> TensorTypeRegistry.find(x).type())
                     .collect(Collectors.toList()),
                 this)

